cypress/e2e/CMS/$BlogSeries/index.js

```diff
@@ -111,6 +124,30 @@ And('blog series description reads {string}', (desc) => {
     cy.get(`p:contains('${desc}')`).should('be.visible');
 });
 
+And('user clicks on {string} sub tab', (contentSubTab) => {
+    cy.get(`ul.admin-list li a:contains("${contentSubTab}")`).click({ force: true });
+});
+
+And('user selects {string} option from Operations for {string}', (option, label) => {
+    cy.get('[data-drupal-selector="edit-vocabularies-cgov-blog-topics"] > :nth-child(4) > .dropbutton-wrapper > .dropbutton-widget > .dropbutton > .list > a').click({ force: true });
```

Comment:
this locator looks very brittle - could you try something like
cy.get(a:contains('${label}').parent().parent().find (a[href*="cgov_blog_topics"]).contains(option).click()

Reply:
Updated